app-modules/bot-discord/src/Events/WelcomeMember.php

```
@@ -0,0 +1,59 @@
+use Discord\WebSockets\Event as Events;
+use He4rt\BotDiscord\Actions\UserCharacterResolver;
+use He4rt\Provider\Enums\ProviderEnum;
+use He4rt\Provider\Models\Provider;
+use He4rt\Tenant\Models\Tenant;
+use Laracord\Events\Event;
+
+class WelcomeMember extends Event
+{
+    protected $handler = Events::GUILD_MEMBER_ADD;
+
+    public function handle(Member $member, Discord $discord): void
```

PHPMD (2.15.0)

Avoid unused parameters such as '$discord'. (undefined)

(UnusedFormalParameter)